pom.xml

```diff
@@ -223,6 +254,15 @@
                 <artifactId>jib-maven-plugin</artifactId>
                 <version>${jib-maven-plugin.version}</version>
                 <configuration>
+                    <container>
+                        <expandClasspathDependencies>true</expandClasspathDependencies>
+                    </container>
+                    <from>
+                        <image>416670754337.dkr.ecr.eu-west-2.amazonaws.com/ci-corretto-build-21:latest</image>
+                    </from>
+                    <to>
+                        <image>169942020521.dkr.ecr.eu-west-2.amazonaws.com/local/document-signing-api:latest</image>
```

Comment:
think it's ok to update this now to the new image: 416670754337.dkr.ecr.eu-west-2.amazonaws.com/document-signing-api:latest